apps/src/music/views/MusicPlayView.tsx

```diff
@@ -0,0 +1,101 @@
+import React, {memo, useCallback, useMemo} from 'react';
+
+import {Button} from '@cdo/apps/componentLibrary/button';
+import {Heading2, BodyTwoText} from '@cdo/apps/componentLibrary/typography';
+import moduleStyles from './music-play-view.module.scss';
+import Link from '@cdo/apps/componentLibrary/link';
+
+import musicI18n from '../locale';
+
+const codeOrgLogo = require(`@cdo/static/code-dot-org-white-logo.svg`);
```

Comment:
We have a module declaration for .svg and .png files in our Globals typescript file, so you should be able to just `import` this